markdownlint.js

```diff
@@ -191,7 +208,8 @@ program
   .option('-d, --dot', 'include files/folders with a dot (for example `.github`)')
   .option('-f, --fix', 'fix basic errors (does not work with STDIN)')
   .option('-i, --ignore [file|directory|glob]', 'file(s) to ignore/exclude', concatArray, [])
-  .option('-o, --output [outputFile]', 'write issues to file (no console)')
+  .option('-j, --json', 'write issues in json format')
+  .option('-o, --output [outputFile]', 'write issues to file (no console)')  
```

Comment:
There are 2 trailing spaces on this line that make it show in the diff.

Reply:
Got it